<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор паролей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>700</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="MainMenu">
     <widget class="QPushButton" name="generateBtn">
      <property name="geometry">
       <rect>
        <x>139</x>
        <y>230</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сгенерировать пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>139</x>
        <y>290</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сохранить пароль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewBtn">
      <property name="geometry">
       <rect>
        <x>139</x>
        <y>350</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Список паролей</string>
      </property>
     </widget>
     <widget class="QLabel" name="headerLabel_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Генератор паролей</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PasswordGeneratorMenu">
     <widget class="QLabel" name="headerLabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Генерация пароля</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="customGenBtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>271</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Настраиваемый генератор</string>
      </property>
     </widget>
     <widget class="QPushButton" name="complexityGenBtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>271</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>По сложности</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtnGeneratorMenu">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>271</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CustomPasswordGenerator">
     <widget class="QLabel" name="headerLabel_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Настраиваемый генератор</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Длина пароля:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lengthInput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="lowerCaseCheck">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Использовать строчные буквы (a-z)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="upperCaseCheck">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Использовать заглавные буквы (A-Z)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="digitsCheck">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Использовать цифры (0-9)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="specialCheck">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>370</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Использовать спецсимволы (!@#$%^&amp;*)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateCustomBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>430</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сгенерировать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtnCustomGenerator">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ComplexityPasswordGenerator">
     <widget class="QLabel" name="headerLabel_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Генератор по сложности</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сложность:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="complexityCombo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Низкая</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Средняя</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Высокая</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="generateComplexityBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сгенерировать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtnComplexityGenerator">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SavePasswordMenu">
     <widget class="QLabel" name="headerLabel_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сохранение пароля</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="showPasswordCheck">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Показать пароль</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ключ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="keyInputSave">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>300</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="showKeySaveCheck">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Показать ключ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="savePasswordBtn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>390</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtnSavePassword">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SavedPasswordsMenu">
     <widget class="QLabel" name="headerLabel_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сохранённые пароли</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Поиск:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="passwordTable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>401</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Логин</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Пароль</string>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>480</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ключ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="keyInputView">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>480</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="showKeyViewCheck">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>510</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Показать ключ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewPasswordBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>550</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Посмотреть</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removePasswordBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>550</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtnViewPasswords">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>610</y>
        <width>421</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
